datalake/src/main/java/com/sequenceiq/datalake/flow/datalake/recovery/DatalakeUpgradeRecoveryActions.java

```diff
@@ -55,6 +64,8 @@ protected SdxContext createFlowContext(FlowParameters flowParameters, StateConte
             @Override
             protected void doExecute(SdxContext context, DatalakeRecoveryStartEvent payload, Map<Object, Object> variables) {
                 LOGGER.info("Datalake recovery has been started for {}", payload.getResourceId());
+                SdxCluster sdxCluster = sdxService.getById(payload.getResourceId());
+                eventSenderService.sendEventAndNotification(sdxCluster, context.getFlowTriggerUserCrn(), ResourceEvent.DATALAKE_RECOVERY_STARTED);
```

Comment:
As I see, there is zero UT coverage for this class. Could you please create one and add some test cases checking at least the new logic changes here?